### PR TITLE
Feature/request router

### PR DESCRIPTION
**Pull request by corbinrobb**
Create replies and threads tables, create seeds for new tables, create request router, add GET and POST endpoints for request router. 

Still needs documentation and testing

### CODE REVIEW

**Review by bobbygondola — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]

**Comment by Vyraal1** [comment]
Mostly looks good to me, some things I have only a small opinion about it (but not worth delaying the PR for)
- for the migrations and table creations, I would normally separate out the files on a per table basis, it makes reversing migrations possibly less painful if needed, instead of needing to drop the entire database since it's only one migration
- some of the table  names are a bit wordy, could be a bit more concise, but this only serves as a minor inconvenience, everything else looks good to me
